src/proto/FunctionRpc.proto

```diff
@@ -289,11 +289,35 @@ message TypedData {
     bytes bytes = 3;
     bytes stream = 4;
     RpcHttp http = 5;
-	  sint64 int = 6;
+    sint64 int = 6;
     double double = 7;
+    TypedDataCollectionBytes collection_bytes = 8;
```

Comment:
Remove prefix TypedData as this is already under TypedData message

Reply:
Sounds good, I just generalized the types to be used in other messages in future.